hub/components/NewWallet/useNewWalletTransaction.ts

```diff
@@ -51,12 +56,14 @@ const useNewWalletCallback = (
     );
 
     const instructions: TransactionInstruction[] = [];
+    const createCreateGovernanceTicketIxs: TransactionInstruction[] = [];
```

Comment:
it could just be called createTicketIxs or whatever. its clear from context we are making a new governance. also adding all this extra code, just for nft-voter, any time we do anything is kinda lame. i think its because you are trying to make ticket creation parallel for efficiency reasons, but im not sure thats even how the thing works because i dont think the sequential ones wait for the parallel ones.

Reply:
however hubs folder is not actually maintained

Reply:
hmm i thought the sequential one will wait for the parallel ones, not very sure as well. Since their document said we can create [parallel, sequential, parallel], and if the sequential won't wait, the original nft-voter castVote might not work well?

Reply:
hmmmmm I don't know sorry.